indico/modules/events/requests/base.py

```diff
@@ -61,6 +61,8 @@ class RequestDefinitionBase(object):
     manager_form = RequestManagerForm
     #: default values to use if there's no existing request
     form_defaults = {}
+    #: whether requests of this type should be automatically accepted
```

Comment:
isn't this misleading since it's still up to the implementation to do the auto-accepting?

Reply:
OK, let's do this in the way we discussed IRL.